templates/foreman/template.yml

```diff
@@ -0,0 +1,11 @@
+skeleton_path: ../skeleton.json
+skeleton_index_pattern_path: ../skeleton-index-pattern.json
+
+elasticsearch_template:
+  name: org.foreman.viaq-cdm
+  index_pattern: "project.foreman-*"
+  order: 20
+
+namespaces:
+  - foreman.yml
+  - systemd.yml
```

Comment:
add `- pipeline_metadata.yml` here - see https://github.com/ViaQ/elasticsearch-templates/blob/master/namespaces/pipeline_metadata.yml for some optional fields you can add to your log records which can help with tracking where and how the records came.